13.0/base_requirements.txt

```
@@ -0,0 +1,86 @@
+# Odoo dependencies
+Babel==2.3.4
+chardet==3.0.4
+decorator==4.0.10
+docutils==0.12
+ebaysdk==2.1.5
+feedparser==5.2.1
+gevent==1.1.2 ; sys_platform != 'win32' and python_version < '3.7'
+greenlet==0.4.10 ; python_version < '3.7'
```

```suggestion
greenlet==0.4.13 ; python_version >= '3.7'
```

Same as gevent, 3.7 is targeted